CMakeLists.txt

```diff
@@ -17,12 +17,13 @@ cmake_policy(SET CMP0025 NEW)
 cmake_policy(SET CMP0048 NEW)
 cmake_policy(SET CMP0057 NEW)
 
-project(Umpire 
+project(Umpire
   LANGUAGES CXX C
   VERSION 0.3.0)
 
 set(ENABLE_OPENMP On CACHE Bool "")
 set(ENABLE_CUDA On CACHE Bool "")
+set(ENABLE_NUMA Off CACHE Bool "")
```

Comment:
@jdahm as discussed, we should consider disabling CUDA resources when ENABLE_NUMA is on. This might be best done with another option.

Is there a way to only consider CPU attached NUMA nodes?

Reply:
I still think this is a good idea, we should have 3 possible configurations:
1. NUMA everywhere (CPU & GPU)
2. NUM CPU, CUDA for GPU
3. No NUMA